tool/tsh/tsh.go

```diff
@@ -600,8 +603,10 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		EnumVar(&cf.MFAMode, modes...)
 	app.HelpFlag.Short('h')
 
-	ver := app.Command("version", "Print the version of your tsh binary")
+	ver := app.Command("version", "Print the version of your tsh binary and version of the Teleport proxy in current tsh profile")
 	ver.Flag("format", defaults.FormatFlagDescription(defaults.DefaultFormats...)).Short('f').Default(teleport.Text).EnumVar(&cf.Format, defaults.DefaultFormats...)
+	ver.Flag("client", "Only display the version of the tsh client. Does not connect to a Teleport proxy").
```

Comment:
```suggestion
	ver.Flag("client", "Show the client version only (no server required).").
```

Similarly, to match kubectl.